extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -36,7 +35,6 @@ import {toggle} from '../../../src/style';
 installActivityService(AMP.win);
 installCidService(AMP.win);
 installCryptoService(AMP.win);
-installStorageService(AMP.win);
```

Comment:
We gonna get a bit of a version skew. @cramforce is this ok? Or should we do double-installation for one cycle?